apps/api/src/pkg/errors/http.ts

```diff
@@ -13,7 +13,7 @@ const ErrorCode = z.enum([
   "USAGE_EXCEEDED",
   "DISABLED",
   "NOT_FOUND",
-  "NOT_UNIQUE",
+  "CONFLICT",
```

Comment:
I adjusted it FROM NOT_UNIQUE to CONFLICT as this better fits the 409 status code.